components/prize_pool/commons/prize_pool.lua

```diff
@@ -988,7 +988,7 @@ function Placement:_parseOpponentArgs(input, date)
 
 	local opponentData = Opponent.readOpponentArgs(opponentArgs)
 
-	if not opponentData or Opponent.isTbd(opponentData) then
+	if not opponentData or (opponentData.type ~= Opponent.literal and Opponent.isTbd(opponentData)) then
```

Comment:
```suggestion
	if not opponentData or (Opponent.isTbd(opponentData) and opponentData.type ~= Opponent.literal) then
```
Only personal preference, as isTbd is the main thing in question, literal being the exception